src/data.js

```diff
@@ -1,21 +1,40 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-alert */
 /* eslint-disable import/prefer-default-export */
-export const data = [
-  {
-    index: 0,
-    description: 'Set up the project',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'Design the home and about pages',
-    completed: false,
-  },
-  {
-    index: 2,
-    description: 'Fix linter errors',
-    completed: false,
-  },
-];
+
+class Task {
+  constructor(description) {
+    this.description = description;
+    this.index = new Date();
+    this.completed = false;
+  }
+}
+
+const addTaskForm = document.querySelector('#add-task');
+const data = JSON.parse(localStorage.getItem('Task-list')) || [];
+addTaskForm.addEventListener('submit', () => {
+  const inputTaskValue = document.querySelector('#add-task-input').value;
+  if (inputTaskValue === '') return;
+  const newTask = new Task(inputTaskValue);
+  data.push(newTask);
+  localStorage.setItem('Task-list', JSON.stringify(data));
+});
+
+const clearTask = document.querySelector('.clear-task');
+
+clearTask.addEventListener('click', () => {
+  if (data.length > 0) {
+    localStorage.clear('Task-list');
+    location.reload();
+  } else {
+    alert('Local storage is empty');
```

Comment:
## [Optional]

This should be production code, we shouldn't have `alert`, please remove the `alert`.